decidim-meetings/app/views/decidim/meetings/meetings/show.html.erb

```diff
@@ -166,6 +166,10 @@ edit_link(
       </div>
     <% end %>
 
+    <% if current_user && meeting.live? %>
+      <div class="timeout-prevention hide" data-prevent-for="<%= (meeting.end_time - Time.current).to_i %>"></div>
```

Comment:
I would rather pass this as an option in the `#timeoutModal` element where we are already passing other options. These can be generally confusing when the settings are scattered in multiple places.

Maybe we can add a helper method for getting the prevention time and then override it in the meetings controller?

Reply:
Yeah that's better